.github/workflows/quicktest.yml

```diff
@@ -30,16 +30,12 @@ jobs:
             phpcs_version: 'dev-master'
             wpcs_version: '2.3.*'
           - php: '5.4'
-            phpcs_version: '3.5.5'
+            phpcs_version: '3.7.1'
             wpcs_version: '2.3.*'
 
           - php: 'latest'
             phpcs_version: 'dev-master'
             wpcs_version: '2.3.*'
-          - php: 'latest'
-            # PHPCS 3.6.1 is the lowest version of PHPCS which supports PHP 8.1.
-            phpcs_version: '3.6.1'
-            wpcs_version: '2.3.*'
```

Comment:
Why are you removing this build ?

The "Quick test" workflow was set up to run the tests against the lowest and highest supported PHP/PHPCS combinations. The build you are removing is the "high PHP/low PHPCS".

I suggest bringing it back, removing the comment and changing the `phpcs_version` to `3.7.1` instead.